generators/chipyard/src/main/scala/config/RoCCAcceleratorConfigs.scala

```diff
@@ -67,3 +67,9 @@ class ReRoCCManyGemminiConfig extends Config(
   new gemmini.LeanGemminiConfig ++                              // rerocc tile0 is gemmini
   new freechips.rocketchip.subsystem.WithNBigCores(4) ++        // 4 rocket cores
   new chipyard.config.AbstractConfig)
+
+class ZstdCompressorRocketConfig extends Config(
+  new compressacc.WithZstdCompressor ++
+  new chipyard.config.WithSystemBusWidth(256) ++
```

Comment:
Is this required for the implementation to work? Can't you just WidthWidget in the implementation?